src/main/java/sk/uniba/fmph/dcs/game_board/EvaluateCivilisationCardImmediateEffect.java

```diff
@@ -1,7 +1,8 @@
-package sk.uniba.fmph.dcs.game_board;
-
-import sk.uniba.fmph.dcs.stone_age.Effect;
-
-public interface EvaluateCivilisationCardImmediateEffect {
-    boolean performEffect(Player player, Effect choice);
-}
+package sk.uniba.fmph.dcs.game_board;
+
+import sk.uniba.fmph.dcs.stone_age.ActionResult;
+import sk.uniba.fmph.dcs.stone_age.Effect;
+
+public interface EvaluateCivilisationCardImmediateEffect {
+    ActionResult performEffect(Player player, Effect choice);
+}

```

Comment:
```suggestion
import java.util.Collection;

public interface EvaluateCivilisationCardImmediateEffect {
    ActionResult performEffect(Player player, Collection<Effect> choice);
}

```
Nie som si isty ci toto spominal Lukotka, ale osobne to dava ovela vacsi zmysel.

Reply:
Teraz mi doslo, ze to treba tym padom zmenit vsade, preto napisem na discord, ci je to dobry napad lebo potom to menit naspat by bolo zle.